Documentation/ApiOverview/Database/Connection/Index.rst

```diff
@@ -95,19 +95,28 @@ bulkInsert()
    $connection = GeneralUtility::makeInstance(ConnectionPool::class)
       ->getConnectionForTable('sys_log')
    $connection->bulkInsert(
-      'sys_log',
-      [
-         [(int)$userId, (string)$details1],
-         [(int)$userId, (string)$details2],
-      ],
-      [
-         'userid',
-         'details',
-      ],
-      [
-         Connection::PARAM_INT,
-         Connection::PARAM_STR,
-      ]
+       'sys_log',
+       [
+           [
+               (int)$userId,
+               (string)$details1
+           ],
+           [
+               'userId' => (int)$userId,
+               'detail' => (string)$details2
```

Comment:
If I understand you correctly you are trying to show two alternative Strategies of mapping the data to the fields here? That is why array one and array two of the second parameter look so different?

I would suggest to break up the one example into two examples and some explanatory lines of what you are trying to achieve